hardware/NestOAuthAPI.cpp

```diff
@@ -562,7 +562,7 @@ void CNestOAuthAPI::GetMeterDetails()
 
 			// Find out if we're using C or F.
 			std::string temperatureScale = ndevice["temperature_scale"].asString();
-			boost::to_lower(temperatureScale);
+			std::transform(temperatureScale.begin(), temperatureScale.end(), temperatureScale.begin(), ::tolower);
```

Comment:
You could use the helper.c/h function 'stdlower'

Reply:
Will do.